classes/context/Context.php

```diff
@@ -546,6 +546,26 @@ public function isInstitutionStatsEnabled(Site $site): bool
         }
         return (bool) $enableInstitutionUsageStats;
     }
+
+    /**
+    * What Geo data to track for usage statistics.
+    * Consider context setting only if the site setting is enabled and context setting considers less Geo data than site setting.
+    *
+    * @return ?string Possible return values: null, disabled, country, coutnry+region, country+region+city
```

Comment:
`country+region`

Reply:
Is there any difference in how the system work between `null` and `disabled`? I wonder if we should set a value by default and enforce either `disabled` or one of the three options? Seems like we shouldn't need `null` and `disabled`.

Reply:
There is no difference how the system behaves.
I have an extra issue regarding this, that I would like to take a look at with you: https://github.com/pkp/pkp-lib/issues/7976.